app/models/payment_orders/bank_link.rb

```diff
@@ -57,6 +57,18 @@ def complete_transaction
         iban: invoice.seller_iban
       )
 
+      # Transaction already autobinded (possibly) invalid invoice
+      if transaction.binded?
```

Comment:
Looks similar to [this method](https://github.com/internetee/registry/pull/1503/files#diff-e501c6a999b2d55ca6d179a0d68bbadfR40) . Maybe we may move it to, say, concern and use the same method of bank transaction creation in both cases?